fuel-core/src/tx_pool.rs

```diff
@@ -103,7 +116,7 @@ impl TxPool {
                 // TODO: compute the current merkle root of all blocks
                 prev_root: Default::default(),
             },
-            transactions: vec![tx],
+            transactions: txs_to_mine,
```

Comment:
I would avoid using terms like "mine," as it's a heavily-loaded term from a bygone era.

Reply:
Yeah a couple variable names I wasn't sure and probably will change some when I get tests passing. Would you recommend using "include" then?

Reply:
Ye